src/main/java/org/gradle/android/VersionNumber.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright 2012 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.android;
```

Comment:
We took this from Gradle BT. Do we properly mention this?

Reply:
I see. Not sure this is enough or not, given I do not have much legal knowledge. I think we can leave it as it is for now, given all work done here and at the original class, no 3rd-parties are involved.